knexfile.js

```diff
@@ -1,4 +1,4 @@
-import { merge } from 'lodash'
+const merge = require('lodash').merge
```

Comment:
i'm assuming you did this because the migrations script wasn't running babel? my bad. you can also require lodash functions individually, e.g.: `const merge = require('lodash/merge')`